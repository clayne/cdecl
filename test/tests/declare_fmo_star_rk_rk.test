cdecl @ @ @ struct S; declare * as friend member operator (reference to S) returning reference to S @ 65
