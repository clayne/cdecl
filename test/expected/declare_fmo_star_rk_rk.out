declare * as friend member operator (reference to S) returning reference to S
        ^
9: error: operator overloading not supported in C
